src/sort.rs

```diff
@@ -126,14 +187,31 @@ fn sort_array(arr: &mut Array) {
     arr.set_trailing_comma(trailing_comma);
 }
 
-fn sort_table(table: &mut Table, group: bool) {
+fn sort_table(table: &mut Table, group: bool, target_tables: &TargetTablePaths) {
     if group {
         sort_by_group(table);
+    } else if !target_tables.is_empty() {
```

Comment:
This new functionality seems to be entirely unrelated to the `group` functionality. Should we move it to a separate function?